index.js

```diff
@@ -137,18 +148,28 @@ export default class LinterJSCS {
     };
   }
 
+  static getFilePath(path) {
+    const relative = atom.project.relativizePath(path);
+    return relative[1];
+  }
+
+  static getConfig(filePath) {
+    return configFile.load(false,
+      path.join(path.dirname(filePath), this.configPath));
```

Comment:
You should probably just add `atom-linter` to the package deps and use it's `find()` function to search up the tree, this will match how [`jscs`](http://jscs.info/overview#-config-c) handles it.